packages/viewer/src/components/scene-tree-table-header/scene-tree-table-header.tsx

```diff
@@ -0,0 +1,22 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'vertex-scene-tree-table-header',
+  styleUrl: 'scene-tree-table-header.css',
+  shadow: true,
+})
+export class SceneTreeTableHeader {
+  /**
+   * The label to display in this header.
+   */
+  @Prop()
+  public label?: string;
+
+  public render(): h.JSX.IntrinsicElements {
+    return (
+      <Host>
+        <div class="header">{this.label}</div>
```

Comment:
This could also be a slot. Gives a bit more flexibility.

```html
<vertex-scene-tree-table-header>
  <button>my label</button>
</vertex-scene-tree-table-header>
```